app/Models/Post.php

```diff
@@ -346,7 +346,7 @@ public function scopeWhereVisible($query)
      */
     public function scopeWithHiddenPosts($query)
     {
-        if ((! token()->exists()) || (! in_array(token()->role, ['admin', 'staff']))) {
+        if (! is_staff_user()) {
```

Comment:
@DFurnes I changed this to use the helper function here. I initially didn't use the helper function because I thought that the token was that of Phoenix, not the user, so wanted to omit the client check. I think this is more accurate to use!

Reply:
FWIW, the `is_staff_user` helper should work for clients as well!

Reply:
yes! I initially thought we didn't want it to work for clients but now I see I misunderstood ! :)